README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
 
 The state of the ApplicationPool. Must be either `Started` or `Stopped`
 
+#### `auto_start`
+
+When true, indicates to the World Wide Web Publishing Service (W3SVC) that the application pool should be automatically started when it is created or when IIS is started. Valid options `true` or `false`
```

Comment:
values `true` and `false` should be in backticks throughout

Needs a period at the end of the line; this is true throughout the README